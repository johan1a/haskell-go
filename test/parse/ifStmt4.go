package main
import "fmt";
func main() {
if 2 == 2 {
    fmt.Print( 1 )
} else {
    fmt.Print ( 2 )
}
}
