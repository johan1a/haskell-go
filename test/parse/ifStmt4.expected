SourceFile (Package "main") [ImportDecl [I1 "fmt"]] [TopLevelDecl2 (FunctionDecl2 "main" (Signature1 []) (Block [IfStmt (Ifstmt2 (BinExpr (CondExpr (Eq_ (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 2)))))) (UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 2))))))))) (Block [SimpleStmt (ExpressionStmt (UnaryExpr (PrimaryExpr (PrimaryExpr7 (PrimaryExpr1 (Operand2 (OperandName2 (QualifiedIdent "fmt" "Print")))) (Arguments5 [UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 1)))))])))))]) (Else2 (Block [SimpleStmt (ExpressionStmt (UnaryExpr (PrimaryExpr (PrimaryExpr7 (PrimaryExpr1 (Operand2 (OperandName2 (QualifiedIdent "fmt" "Print")))) (Arguments5 [UnaryExpr (PrimaryExpr (PrimaryExpr1 (Operand1 (BasicLit (IntLit 2)))))])))))])))]))]
